scijava/scijava-types/src/main/java/org/scijava/types/TypeTools.java

```diff
@@ -302,6 +304,20 @@ else if (paramComponent instanceof TypeVariable) {
 		return true;
 	}
 
+	public static Type raiseParametersToClass(Class<?> cls, final Class<?> superCls, final Type[] superClsTypeVars) {
```

Comment:
The `superClsTypeVars` could be a `Type...` instead, to make calling code a little more readable.

I would also rename this method to something less bulky, especially since it is called by nearly every `TypeExtractor` implementations as its final step.

Reply:
Renamed in 035e1c1